pkgs/applications/science/misc/motu-client/default.nix

```diff
@@ -0,0 +1,23 @@
+{ pkgs, python27Packages } :
+python27Packages.buildPythonApplication rec {
```

Comment:
I have not tested  that with `pypy`, but I am certain that it is definitely not python3 compatible. I preferred that here to make it clear it will not work with python3 (even if `pythonPackages` is just an alias for `python27Packages` at the moment, I prefer explicit over implicit)